README.md

```diff
@@ -12,9 +12,13 @@ Current Build Status is: [![Build Status](https://secure.travis-ci.org/m-lab/m-l
 2. Run Jekyll server and pass in a blank baseurl to preview in development mode `jekyll serve --baseurl`.
 3. View the generated site by going to [http://localhost:4000/](http://localhost:4000/)
 
+### Pre-commit Hook
+
+Local Developers should also install the pre-commit hook that comes packaged with this repository that will alert about any trailing whitespaces to minimize git diff noise.  In order to install the pre-commit hook, run the following command to create a symbolic link: ``rm -rf .git/hooks/ && ln -s -f ../_hooks .git/``.
```

Comment:
Good catch, changed that to triple now as that seems like that has benefits over single, but don't think it matters in this case.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/2eab78f260fb67321df8d52aa62a71143f1c37ef